ci/travis-36-slow.yaml

```diff
@@ -5,6 +5,7 @@ channels:
 dependencies:
   - beautifulsoup4
   - cython
+  - gcsfs
```

Comment:
so I would remove gcsfs from the *slow* ci files (not tested anyhow as these tests are not slow)

Reply:
Done.